sdk/core/src/services/WalletStorageService.ts

```diff
@@ -102,14 +102,24 @@ export default class WalletStorageService {
     return responses
   }
 
-  public async getAllCredentials(types?: string[][], storageRegion?: string) {
+  public async getAllCredentials(storageRegion?: string) {
     storageRegion = storageRegion || this._storageRegion
-    types = types || []
 
-    const credentials = await this._affinidiVaultStorageService.getAllCredentials(types, storageRegion)
+    const credentials = await this._affinidiVaultStorageService.searchCredentials([], storageRegion)
 
     // should be deleted during migration to affinidi-vault Phase #2
-    const bloomCredentials = await this._bloomVaultStorageService.getAllCredentials(types, storageRegion)
+    const bloomCredentials = await this._bloomVaultStorageService.searchCredentials([], storageRegion)
+
+    return [...credentials, ...bloomCredentials]
+  }
+
+  public async getCredentialsByTypes(types: string[][], storageRegion?: string) {
```

Comment:
Maybe we should make this one private and remove `getCredentialsByTypes` from `CommonNetworkMember`.

(The problem with making more methods public is that even though they might be completely useless, we'll still have to support them and maintain backwards compatibility in case one of the external clients relies on them. So it's better to keep the API surface area as small as possible)